heartbeat/monitors/wrappers/summarizer/plugstatestat.go

```diff
@@ -56,20 +56,24 @@ func (ssp *BrowserStateStatusPlugin) EachEvent(event *beat.Event, jobErr error)
 }
 
 func (ssp *BrowserStateStatusPlugin) BeforeSummary(event *beat.Event) BeforeSummaryActions {
+	ssp.cssp.js.Status = monitorstate.StatusDown
 	if ssp.cssp.js.Down == 0 {
 		// Browsers don't have a prior increment of this, so set it to some
 		// non-zero value
 		ssp.cssp.js.Up = 1
+		ssp.cssp.js.Status = monitorstate.StatusUp
 	}
 
 	res := ssp.cssp.BeforeSummary(event)
-
+	// Browsers don't set this prior, so we set this here, as opposed to lightweight monitors
 	_, _ = event.PutValue("monitor.status", string(ssp.cssp.js.Status))
+
+	_, _ = event.PutValue("synthetics", mapstr.M{"type": "heartbeat/summary"})
```

Comment:
Just cleaner to move this here rather than have the extra branch in the common logic